ufl/finiteelement/restrictedelement.py

```diff
@@ -96,3 +96,7 @@ def restricted_sub_elements(self):
         #        w.r.t. different sub_elements meanings.
         "Return list of restricted sub elements."
         return (self._element,)
+
+    def is_fully_continuous(self):
+        """Return true if the values of this element's basis functions are continuous between elements."""
+        return self._element.is_fully_continuous()
```

Comment:
Is this right? If I take the restriction of an $H^1$ space to the interior, is the resulting space continuous, I would say no, there's nothing pinning down the continuity on the faces and vertices any more.

Reply:
I've changed this to return False